benchmarks/distributed/dataloading.py

```diff
@@ -0,0 +1,379 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Distributed data-loading with tensorclass
+=========================================
+"""
+
+##############################################################################
+# This file provides an example of distributed dataloading with tensorclass.
+# It can be run with :obj:`torchrun`. Here is an example of the slurm config to
+# use, although torchrun is compatible with any other scheduler too:
+#
+#  .. code-block::
+#
+#     #!/bin/bash
```

Comment:
thinking about it, it would be clearer to have a `dataloading.sh` no? It wouldnt hurt to keep it there

Reply:
Yeah, that would be good, and then it's easier to run / test too.